docs/UserGuide.md

```diff
@@ -27,7 +27,7 @@ Wishful Shrinking is a **desktop app for managing your diet, keeping track of yo
 
    * **`recipe`** : Lists all recipes.
 
-   * **`addR`**` n/salad i/lettuce, carrots, olive oil` : Adds a `salad` recipe to Wishful Shrinking.
+   * **`addR`**` n/salad i/lettuce, carrots, olive oil c/40` : Adds a `salad` recipe to Wishful Shrinking.
```

Comment:
should we write the whole proper command?

Reply:
Jia qi changes the way of adding instruction and I not sure how does it look like as that part is not merged to the master yet. The addRecipeCommand UG documentation is base on what I know so far.